src/app/courses/add-courses/courses-add.component.html

```diff
@@ -1,118 +1,132 @@
-<h1>Add Courses</h1>
+<h1 i18n>Add Courses</h1>
 <!-- nvalidate - disable browser's default validators to use our validators -->
 <form [formGroup]="courseForm" (ngSubmit)="onSubmit()" novalidate>
   <div class="row">
     <div class="form-group col-4">
-      <label>Course Title</label>
-      <input type="text" class="form-control" formControlName="courseTitle" required>
+      <mat-form-field>
+        <input matInput i18n-placeholder placeholder="Course Title" formControlName="courseTitle" required>
+      </mat-form-field>
       <planet-form-error-messages [control]="courseForm.controls.courseTitle"></planet-form-error-messages>
     </div>
     <div class="form-group col-4">
-      <label>Language of Instruction</label>
-      <input type="text" class="form-control" formControlName="languageOfInstruction">
+      <mat-form-field>
+        <input matInput i18n-placeholder placeholder="Language of Instruction" formControlName="languageOfInstruction" required>
+      </mat-form-field>
```

Comment:
where is form-error-message?

Reply:
there wasnt error message for it previously, if we need we can keep this, this is actually optional field.